src/spi.rs

```diff
@@ -53,19 +56,27 @@ impl Spi<SPI0> {
         // enable APB0 bus
         apb0.enable();
```

Comment:
I noticed that `SPI0` actually required `APB2` enabled and not `APB0` according to this [file](https://github.com/kendryte/kendryte-standalone-sdk/blob/develop/lib/drivers/sysctl.c) from K210 C++ driver. I saw this since before this PR. It's just something that worth noting down.